main/pipeline/pipeline.cc

```diff
@@ -1264,4 +1264,43 @@ vector<ast::ParsedFile> autogenWriteCacheFile(const core::GlobalState &gs, const
 #endif
 }
 
+void printUntypedBlames(const core::GlobalState &gs, const UnorderedMap<long, long> untypedBlames,
+                        const std::string &blameFilePath) {
+#ifndef SORBET_REALMAIN_MIN
+
+    rapidjson::StringBuffer result;
+    rapidjson::Writer<rapidjson::StringBuffer> writer(result);
+
+    writer.StartArray();
+
+    for (auto e : untypedBlames) {
```

Comment:
```suggestion
    for (auto &[symId, count] : untypedBlames) {
```

I have a slight preference for this iteration style, so that people don't have to remember what `.first` and `.second` are. But that's completely non blocking.